js/documents.js

```diff
@@ -527,7 +527,7 @@ var documentsMain = {
 								var filename = path.substring(path.lastIndexOf('/') + 1);
 								$.ajax({
 									type: 'POST',
-									url: OC.linkTo('richdocuments', 'assets'),
+									url: OC.generateUrl('apps/richdocuments/assets'),
```

Comment:
What if it is installed in `apps2`? 🤔

Reply:
Ah shit. see that is why we used the LinkTo...

but LinkTo doesn't take index.php into account

Reply:
Mmmm ok lets fix that in a different PR
https://github.com/nextcloud/richdocuments/blob/4a30187fc11655a5f3de15f91e20f1c90d8a90e6/js/documents.js#L138

We do this in many places